claude-instructions/.claude/skills/jhack/README.md

```diff
@@ -0,0 +1,233 @@
+# Jhack Agent Skill
+
+An expert Claude skill for Juju charm development using jhack utilities.
+
+## Overview
+
+This skill provides comprehensive guidance for using jhack, a collection of developer utilities that streamline Juju charm development, debugging, and testing workflows.
+
+## What This Skill Provides
+
+- **Rapid development workflows**: Sync code and test in seconds
+- **Debugging utilities**: Inspect relations, state, and events
+- **Event manipulation**: Fire, capture, and replay events
+- **Testing integration**: Scenario snapshots and chaos testing
+- **Complete command reference**: All jhack commands documented
+- **Troubleshooting guide**: Solutions for common issues
+
+## Installation
```

Comment:
This whole section should be replaced with a link to the INSTALL-SKILLS.md file.